tensorflow/python/ops/math_ops.py

```diff
@@ -205,6 +215,16 @@ def argmin_v2(input,
 
   Returns:
     A `Tensor` of type `output_type`.
+	
+  Usage:
+    ```python
```

Comment:
This line should start on the same column as `:` above

Reply:
I don't know what's happening. I had made the same indentation for both. Now made the changes